From efd00f8fe8936365ed35942bf5ad17e5f898eb8d Mon Sep 17 00:00:00 2001
From: Peter Somogyi <psomogyi@cloudera.com>
Date: Fri, 7 Jul 2017 13:54:41 +0200
Subject: [PATCH] HBASE-18332 Upgrade asciidoctor-maven-plugin

Update asciidoctor-maven-plugin to 1.5.5 and asciidoctorj-pdf to 1.5.0-alpha.15
asciidoctor's pdfmark generation is turned off.

Change-Id: I6e019807386062719ff0cdb5526778f44daf086f
---
 pom.xml | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/pom.xml b/pom.xml
index 9a554fd4a6..ce2f632b94 100644
--- a/pom.xml
+++ b/pom.xml
@@ -1264,7 +1264,6 @@
             <configuration>
               <target name="rename file">
                 <move file="${project.reporting.outputDirectory}/book.pdf" tofile="${project.reporting.outputDirectory}/apache_hbase_reference_guide.pdf" />
-                <move file="${project.reporting.outputDirectory}/book.pdfmarks" tofile="${project.reporting.outputDirectory}/apache_hbase_reference_guide.pdfmarks" />
               </target>
             </configuration>
             <goals>
@@ -1390,8 +1389,8 @@
     <curator.version>2.12.0</curator.version>
     <!-- Plugin Dependencies -->
     <apache.rat.version>0.12</apache.rat.version>
-    <asciidoctor.plugin.version>1.5.2.1</asciidoctor.plugin.version>
-    <asciidoctorj.pdf.version>1.5.0-alpha.6</asciidoctorj.pdf.version>
+    <asciidoctor.plugin.version>1.5.5</asciidoctor.plugin.version>
+    <asciidoctorj.pdf.version>1.5.0-alpha.15</asciidoctorj.pdf.version>
     <build.helper.maven.version>3.0.0</build.helper.maven.version>
     <buildnumber.maven.version>1.4</buildnumber.maven.version>
     <checkstyle.version>6.18</checkstyle.version>
-- 
2.13.1

